Test fails after upgrading to latest jQuery

I'm on Play 2.4.2 using jQuery 2.1.4 and I get the same error as #2607:

```
[error] Test IntegrationTest.test failed: org.openqa.selenium.WebDriverException: com.gargoylesoftware.htmlunit.ScriptException: TypeError: Cannot find function addEventListener in object [object HTMLDocument]. (http://localhost:3333/assets/js/vendor/jquery.js#24)
[error] Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
[error] System info: host: 'gimel', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.19.0-26-generic', java.version: '1.8.0_45-internal'
```

Updating to the newest htmlunit does not help. In fact, updating to htmlunit > 2.16 does not work at all:

```
[error] Test IntegrationTest.test failed: java.lang.NoClassDefFoundError: org/apache/http/cookie/CommonCookieAttributeHandler, took 2.104 sec
[error]     at com.gargoylesoftware.htmlunit.httpclient.HtmlUnitBrowserCompatCookieSpec.<init>(HtmlUnitBrowserCompatCookieSpec.java:101)
```

Is there any place where we can see which jQuery versions are compatible with which htmlunit versions, and which htmlunit versions are compatible with which Play versions?

FWIW, changing the browser version to chrome still yields errors for me on jquery 2.2.0

[error] - com.gargoylesoftware.htmlunit.javascript.StrictErrorReporter - runtimeError: message=[An invalid or illegal selector was specified (selector: '*,:x' error: Invalid selector: *:x).] sourceName=[http://localhost:19001/assets/lib/jquery/jquery.min.js] line=[2] lineSource=[null] lineOffset=[0]
[info] - com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine - Caught script exception
com.gargoylesoftware.htmlunit.ScriptException: TypeError: Cannot find function createHTMLDocument in object [object DOMImplementation]. (http://localhost:19001/assets/lib/jquery/jquery.min.js#4)
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$HtmlUnitContextAction.run(JavaScriptEngine.java:705) [htmlunit-2.15.jar:2.15]
	at net.sourceforge.htmlunit.corejs.javascript.Context.call(Context.java:620) [htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ContextFactory.call(ContextFactory.java:513) [htmlunit-core-js-2.15.jar:na]
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine.execute(JavaScriptEngine.java:591) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HtmlPage.loadExternalJavaScriptFile(HtmlPage.java:1078) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HtmlScript.executeScriptIfNeeded(HtmlScript.java:393) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HtmlScript$3.execute(HtmlScript.java:274) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HtmlScript.onAllChildrenAddedToPage(HtmlScript.java:288) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HTMLParser$HtmlUnitDOMBuilder.endElement(HTMLParser.java:741) [htmlunit-2.15.jar:2.15]
	at org.apache.xerces.parsers.AbstractSAXParser.endElement(Unknown Source) [xercesImpl-2.11.0.jar:na]
	at com.gargoylesoftware.htmlunit.html.HTMLParser$HtmlUnitDOMBuilder.endElement(HTMLParser.java:701) [htmlunit-2.15.jar:2.15]
	at org.cyberneko.html.HTMLTagBalancer.callEndElement(HTMLTagBalancer.java:1170) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.HTMLTagBalancer.endElement(HTMLTagBalancer.java:1072) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.filters.DefaultFilter.endElement(DefaultFilter.java:206) [nekohtml-1.9.21.jar:na]
	at org.cyberneko.html.filters.NamespaceBinder.endElement(NamespaceBinder.java:330) [nekohtml-1.9.21.jar:na]
	at org.cyberneko.html.HTMLScanner$ContentScanner.scanEndElement(HTMLScanner.java:3126) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.HTMLScanner$ContentScanner.scan(HTMLScanner.java:2093) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.HTMLScanner.scanDocument(HTMLScanner.java:920) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.HTMLConfiguration.parse(HTMLConfiguration.java:499) [nekohtml-1.9.21.jar:1.9.21]
	at org.cyberneko.html.HTMLConfiguration.parse(HTMLConfiguration.java:452) [nekohtml-1.9.21.jar:1.9.21]
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) [xercesImpl-2.11.0.jar:na]
	at com.gargoylesoftware.htmlunit.html.HTMLParser$HtmlUnitDOMBuilder.parse(HTMLParser.java:965) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HTMLParser.parse(HTMLParser.java:247) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.html.HTMLParser.parseHtml(HTMLParser.java:193) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.DefaultPageCreator.createHtmlPage(DefaultPageCreator.java:268) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.DefaultPageCreator.createPage(DefaultPageCreator.java:156) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.WebClient.loadWebResponseInto(WebClient.java:468) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:342) [htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:407) [htmlunit-2.15.jar:2.15]
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.get(HtmlUnitDriver.java:480) [selenium-htmlunit-driver-2.44.0.jar:na]
	at org.openqa.selenium.htmlunit.HtmlUnitDriver.get(HtmlUnitDriver.java:469) [selenium-htmlunit-driver-2.44.0.jar:na]
	at org.fluentlenium.core.Fluent.goTo(Fluent.java:336) [fluentlenium-core-0.10.3.jar:na]
	at com.adagility.dashboard.HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1$$anon$1.delayedEndpoint$com$adagility$dashboard$HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1$$anon$1$1(HostPartnerControllerSpec.scala:15) [it-classes/:na]
	at com.adagility.dashboard.HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1$$anon$1$delayedInit$body.apply(HostPartnerControllerSpec.scala:12) [it-classes/:na]
	at scala.Function0$class.apply$mcV$sp(Function0.scala:34) [scala-library-2.11.7.jar:1.0.0-M1]
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12) [scala-library-2.11.7.jar:1.0.0-M1]
	at org.specs2.mutable.Around$$anonfun$delayedInit$1$$anonfun$apply$1.apply(Around.scala:16) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.mutable.Around$$anonfun$delayedInit$1$$anonfun$apply$1.apply(Around.scala:16) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$class.execute(ResultExecution.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$.execute(ResultExecution.scala:120) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.Result$$anon$10.asResult(Result.scala:227) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$.apply(AsResult.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$$anonfun$effectively$1.apply(AsResult.scala:28) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$$anonfun$effectively$1.apply(AsResult.scala:28) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$class.effectively(ResultExecution.scala:41) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$.effectively(ResultExecution.scala:120) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$.effectively(AsResult.scala:28) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at play.api.test.WithBrowser$$anonfun$around$4.apply(Specs.scala:89) [play-specs2_2.11-2.4.6.jar:2.4.6]
	at play.api.test.WithBrowser$$anonfun$around$4.apply(Specs.scala:89) [play-specs2_2.11-2.4.6.jar:2.4.6]
	at play.api.test.PlayRunners$class.running(Helpers.scala:56) [play-test_2.11-2.4.6.jar:2.4.6]
	at play.api.test.Helpers$.running(Helpers.scala:363) [play-test_2.11-2.4.6.jar:2.4.6]
	at play.api.test.WithBrowser.around(Specs.scala:89) [play-specs2_2.11-2.4.6.jar:2.4.6]
	at org.specs2.mutable.Around$$anonfun$delayedInit$1.apply(Around.scala:16) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.mutable.Around$$anonfun$delayedInit$1.apply(Around.scala:16) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$class.effectively(ResultExecution.scala:41) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$.effectively(ResultExecution.scala:120) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.mutable.Around$class.delayedInit(Around.scala:16) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at play.api.test.WithBrowser.delayedInit(Specs.scala:72) [play-specs2_2.11-2.4.6.jar:2.4.6]
	at com.adagility.dashboard.HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1$$anon$1.<init>(HostPartnerControllerSpec.scala:12) [it-classes/:na]
	at com.adagility.dashboard.HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1.apply(HostPartnerControllerSpec.scala:12) [it-classes/:na]
	at com.adagility.dashboard.HostPartnerControllerSpec$$anonfun$1$$anonfun$apply$1.apply(HostPartnerControllerSpec.scala:12) [it-classes/:na]
	at org.specs2.matcher.Scope$$anon$3$$anonfun$asResult$1.apply(ThrownExpectations.scala:139) [specs2-matcher_2.11-3.6.4.jar:3.6.4]
	at org.specs2.matcher.Scope$$anon$3$$anonfun$asResult$1.apply(ThrownExpectations.scala:139) [specs2-matcher_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$class.execute(ResultExecution.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$.execute(ResultExecution.scala:120) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.Result$$anon$10.asResult(Result.scala:227) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$.apply(AsResult.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.matcher.Scope$$anon$3.asResult(ThrownExpectations.scala:139) [specs2-matcher_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$.apply(AsResult.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.main.CommandLineAsResult$$anon$1.asResult(CommandLineAsResult.scala:17) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.main.CommandLineAsResult$$anonfun$apply$1.apply(CommandLineAsResult.scala:21) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.main.CommandLineAsResult$$anonfun$apply$1.apply(CommandLineAsResult.scala:21) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.dsl.mutable.ExampleDsl1$BlockExample$$anonfun$$greater$greater$1.apply(ExampleDsl.scala:39) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.dsl.mutable.ExampleDsl1$BlockExample$$anonfun$$greater$greater$1.apply(ExampleDsl.scala:39) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$withEnv$1$$anonfun$apply$3.apply(Execution.scala:132) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$class.execute(ResultExecution.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.ResultExecution$.execute(ResultExecution.scala:120) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.Result$$anon$10.asResult(Result.scala:227) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.execute.AsResult$.apply(AsResult.scala:25) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$withEnv$1.apply(Execution.scala:132) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$withEnv$1.apply(Execution.scala:132) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$execute$2$$anonfun$apply$2.apply(Execution.scala:73) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$execute$2$$anonfun$apply$2.apply(Execution.scala:73) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution.setResult(Execution.scala:81) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$execute$2.apply(Execution.scala:73) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Execution$$anonfun$execute$2.apply(Execution.scala:72) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at scala.Option.fold(Option.scala:158) [scala-library-2.11.7.jar:1.0.0-M1]
	at org.specs2.specification.core.Execution.execute(Execution.scala:72) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$executeFragment$1$$anonfun$apply$15.apply(Executor.scala:146) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$executeFragment$1$$anonfun$apply$15.apply(Executor.scala:144) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.core.Fragment.updateExecution(Fragment.scala:49) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$executeFragment$1.apply(Executor.scala:144) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$executeFragment$1.apply(Executor.scala:143) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1$$anonfun$executedFragment$lzycompute$1$1.apply(Executor.scala:122) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1$$anonfun$executedFragment$lzycompute$1$1.apply(Executor.scala:122) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at scalaz.concurrent.Task$$anonfun$delay$1.apply(Task.scala:271) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$delay$1.apply(Task.scala:271) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1$$anonfun$4.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1$$anonfun$4.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$.Try(Task.scala:386) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.step(Future.scala:109) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.listen(Future.scala:75) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.runAsync(Future.scala:142) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.run(Future.scala:159) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task.run(Task.scala:96) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1.executedFragment$lzycompute$1(Executor.scala:122) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1.org$specs2$specification$process$DefaultExecutor$class$$anonfun$$executedFragment$1(Executor.scala:122) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1$$anonfun$apply$9.apply(Executor.scala:128) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.specification.process.DefaultExecutor$$anonfun$sequencedExecution$1$$anonfun$apply$9.apply(Executor.scala:128) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at scalaz.concurrent.Task$$anonfun$delay$1.apply(Task.scala:271) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$delay$1.apply(Task.scala:271) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1$$anonfun$4.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1$$anonfun$4.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$.Try(Task.scala:386) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task$$anonfun$suspend$1.apply(Task.scala:279) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.step(Future.scala:110) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.listen(Future.scala:75) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future$$anon$1$$anonfun$reduceUnordered$2$$anonfun$apply$10.apply(Future.scala:309) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future$$anon$1$$anonfun$reduceUnordered$2$$anonfun$apply$10.apply(Future.scala:308) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scala.collection.Iterator$class.foreach(Iterator.scala:742) [scala-library-2.11.7.jar:1.0.0-M1]
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1194) [scala-library-2.11.7.jar:1.0.0-M1]
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72) [scala-library-2.11.7.jar:1.0.0-M1]
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54) [scala-library-2.11.7.jar:1.0.0-M1]
	at scalaz.concurrent.Future$$anon$1$$anonfun$reduceUnordered$2.apply(Future.scala:308) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future$$anon$1$$anonfun$reduceUnordered$2.apply(Future.scala:304) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.listen(Future.scala:79) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.runAsync(Future.scala:142) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Future.run(Future.scala:159) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at scalaz.concurrent.Task.attemptRun(Task.scala:103) [scalaz-concurrent_2.11-7.1.3.jar:na]
	at org.specs2.control.ActionT$.fromTask(ActionT.scala:125) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.ActionTSupport$class.fromTask(ActionT.scala:188) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.package$Actions$.fromTask(package.scala:27) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.reporter.Reporter$$anonfun$report$1.apply(Reporter.scala:47) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.reporter.Reporter$$anonfun$report$1.apply(Reporter.scala:37) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.runner.SbtRunner$$anonfun$specificationRun$1$$anonfun$2.apply(SbtRunner.scala:72) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.runner.SbtRunner$$anonfun$specificationRun$1$$anonfun$2.apply(SbtRunner.scala:72) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.ActionT$$anonfun$flatMap$1$$anonfun$apply$2.apply(ActionT.scala:36) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.ActionT$$anonfun$flatMap$1$$anonfun$apply$2.apply(ActionT.scala:36) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.StatusT$$anonfun$flatMap$1.apply(StatusT.scala:20) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at org.specs2.control.StatusT$$anonfun$flatMap$1.apply(StatusT.scala:19) [specs2-common_2.11-3.6.4.jar:3.6.4]
	at scala.Function1$$anonfun$andThen$1.apply(Function1.scala:52) [scala-library-2.11.7.jar:1.0.0-M1]
	at scalaz.WriterT$$anonfun$flatMapF$1.apply(WriterT.scala:63) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.WriterT$$anonfun$flatMapF$1.apply(WriterT.scala:62) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.effect.IO$$anonfun$flatMap$1$$anonfun$apply$9.apply(IO.scala:62) [scalaz-effect_2.11-7.1.3.jar:na]
	at scalaz.effect.IO$$anonfun$flatMap$1$$anonfun$apply$9.apply(IO.scala:61) [scalaz-effect_2.11-7.1.3.jar:na]
	at scalaz.Free$$anonfun$flatMap$1$$anonfun$apply$1.apply(Free.scala:60) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.Free$$anonfun$flatMap$1$$anonfun$apply$1.apply(Free.scala:60) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.Free.resume(Free.scala:72) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.Free.go2$1(Free.scala:118) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.Free.go(Free.scala:122) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.Free.run(Free.scala:172) [scalaz-core_2.11-7.1.3.jar:na]
	at scalaz.effect.IO$class.unsafePerformIO(IO.scala:22) [scalaz-effect_2.11-7.1.3.jar:na]
	at scalaz.effect.IOFunctions$$anon$6.unsafePerformIO(IO.scala:227) [scalaz-effect_2.11-7.1.3.jar:na]
	at org.specs2.runner.SbtRunner$$anonfun$newTask$1$$anon$4.execute(SbtRunner.scala:37) [specs2-core_2.11-3.6.4.jar:3.6.4]
	at sbt.TestRunner.runTest$1(TestFramework.scala:76) [testing-0.13.8.jar:0.13.8]
	at sbt.TestRunner.run(TestFramework.scala:85) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:185) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202) [testing-0.13.8.jar:0.13.8]
	at sbt.TestFunction.apply(TestFramework.scala:207) [testing-0.13.8.jar:0.13.8]
	at sbt.Tests$.sbt$Tests$$processRunnable$1(Tests.scala:239) [actions-0.13.8.jar:0.13.8]
	at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245) [actions-0.13.8.jar:0.13.8]
	at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245) [actions-0.13.8.jar:0.13.8]
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44) [task-system-0.13.8.jar:0.13.8]
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44) [task-system-0.13.8.jar:0.13.8]
	at sbt.std.Transform$$anon$4.work(System.scala:63) [task-system-0.13.8.jar:0.13.8]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226) [tasks-0.13.8.jar:0.13.8]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226) [tasks-0.13.8.jar:0.13.8]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17) [control-0.13.8.jar:0.13.8]
	at sbt.Execute.work(Execute.scala:235) [tasks-0.13.8.jar:0.13.8]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226) [tasks-0.13.8.jar:0.13.8]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226) [tasks-0.13.8.jar:0.13.8]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159) [tasks-0.13.8.jar:0.13.8]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28) [tasks-0.13.8.jar:0.13.8]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_66]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_66]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_66]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_66]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_66]
Caused by: net.sourceforge.htmlunit.corejs.javascript.EcmaError: TypeError: Cannot find function createHTMLDocument in object [object DOMImplementation]. (http://localhost:19001/assets/lib/jquery/jquery.min.js#4)
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3629) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3613) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.typeError(ScriptRuntime.java:3634) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.typeError2(ScriptRuntime.java:3650) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.notFunctionError(ScriptRuntime.java:3714) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.getPropFunctionAndThisHelper(ScriptRuntime.java:2233) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.getPropFunctionAndThis(ScriptRuntime.java:2215) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.Interpreter.interpretLoop(Interpreter.java:1333) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.Interpreter.interpret(Interpreter.java:798) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.InterpretedFunction.call(InterpretedFunction.java:105) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.ContextFactory.doTopCall(ContextFactory.java:411) [htmlunit-core-js-2.15.jar:na]
	at com.gargoylesoftware.htmlunit.javascript.HtmlUnitContextFactory.doTopCall(HtmlUnitContextFactory.java:309) ~[htmlunit-2.15.jar:2.15]
	at net.sourceforge.htmlunit.corejs.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3057) ~[htmlunit-core-js-2.15.jar:na]
	at net.sourceforge.htmlunit.corejs.javascript.InterpretedFunction.exec(InterpretedFunction.java:115) ~[htmlunit-core-js-2.15.jar:na]
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$3.doRun(JavaScriptEngine.java:582) ~[htmlunit-2.15.jar:2.15]
	at com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine$HtmlUnitContextAction.run(JavaScriptEngine.java:690) [htmlunit-2.15.jar:2.15]
	... 184 common frames omitted
[info] - com.gargoylesoftware.htmlunit.javascript.JavaScriptEngine - Caught script exception
com.gargoylesoftware.htmlunit.ScriptException: Error: Bootstrap's JavaScript requires jQuery (http://localhost:19001/assets/lib/bootstrap/js/bootstrap.min.js#6)

--- a/framework/project/Dependencies.scala
+++ b/framework/project/Dependencies.scala
@@ -264,6 +264,7 @@ object Dependencies {
     junitInterface,
     guava,
     findBugs,
+    "net.sourceforge.htmlunit" % "htmlunit" % "2.20", // adds support for jQuery 2.20; can be removed as soon as fluentlenium has it in it's own dependencies
     ("org.fluentlenium" % "fluentlenium-core" % "0.10.9")
       .exclude("org.jboss.netty", "netty")
   )

--- a/templates/play-java-intro/test/IntegrationTest.java
+++ b/templates/play-java-intro/test/IntegrationTest.java
@@ -20,7 +20,6 @@ public class IntegrationTest {
     @Test
     public void test() {
         running(testServer(3333, fakeApplication(inMemoryDatabase())), HTMLUNIT, browser -> {
-            ((HtmlUnitDriver) browser.getDriver()).setJavascriptEnabled(false);
             browser.goTo("http://localhost:3333");
             assertThat(browser.pageSource(), containsString("Add Person"));
         });

--- a/templates/play-scala-intro/test/IntegrationSpec.scala
+++ b/templates/play-scala-intro/test/IntegrationSpec.scala
@@ -16,7 +16,7 @@ class IntegrationSpec extends Specification {
 
   "Application" should {
 
-    "work from within a browser" in new WithBrowser(webDriver = new HtmlUnitDriver(false)) {
+    "work from within a browser" in new WithBrowser {
 
       browser.goTo("http://localhost:" + port)
 

